project-backend/Models/JobController/GetJobBids/GetJobBidsResponseObject.cs

```diff
@@ -0,0 +1,30 @@
+﻿using project_backend.Models.Bid;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace project_backend.Models.JobController.GetJobBids
+{
+    public class GetJobBidsResponseObject
+    {
+        public List<JobBidDTO> Bids { get; set; }
+        public GetJobBidsResponseObject(IList<BidDAO> bidsInput)
+        {
+            Bids = bidsInput.Select(b => new JobBidDTO
+            {
+                Id = b.Id,
+                JobId = b.JobId,
+                Sum = b.Sum,
+                WorkerId = b.WorkerId
+            }
+            ).ToList();
+        }
+    }
+
+    public class JobBidDTO
+    {
+        public int Id { get; set; }
+        public float Sum { get; set; }
+        public int WorkerId { get; set; }
```

Comment:
Instead of returning the WorkerId I guess we can return a WorkerDTO, that has the image, first name and last name, since that will be for sure displayed in the list of bids, and making 20 other requests for each worker wouldn't be that nice. You can add other info that you think might be useful as well to that worker dto

Reply:
Changed it so that we return the name and the id. The image isn't part of the user yet so that should be added later on.